rosserial_tivac/src/rosserial_tivac/make_libraries

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
```

Comment:
Ah, I didn't realise they were not the same. In that case, can you document (perhaps in the usage strings, which should be updated anyway) that the two are different because they are serving these slightly differing purposes?